bin/gftools-fix-font.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+gftools fix-font
+
+Update a font so it conforms to the Google Fonts specification
+https://github.com/googlefonts/gf-docs/tree/master/Spec
+"""
+import argparse
+import logging
+from fontTools.ttLib import TTFont
+from gftools.fix import *
+
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("font", help="Path to font")
+    parser.add_argument("-o", "--out", help="Output path for fixed font")
+    parser.add_argument("--hotfix", action="store_true", help="Hotfix fonts.")
```

Comment:
I *think* I know what this is doing (applying a certain genre of fixes that *should* be fixed in the sources in addition to the genre of fixes that are rightly post-build cleanup) but the argument itself and help text are kind of misleading. My first impression of `--hotfix` would be that the fixes were being applied inline (i.e. obsoleting the `--out` argument). Is this an established name/concept elsewhere in the tool set or is there perhaps a better name for this?

Reply:
> I think I know what this is doing (applying a certain genre of fixes that really ought be fixed in the sources in addition to the genre of fixes that are rightly post-build cleanup and can't be corrected in the sources)

Correct.

> but the argument itself and help text are kind of misleading.

Agreed. I still haven't thought of a decent name for this. First thought was `--force`. Open to suggestions?

> My first impression of --hotfix would be that the fixes were being applied inline (i.e. obsoleting the --out argument). Is this an established name/concept elsewhere in the tool set or is there perhaps a better name for this?

I don't think we have any scripts which inline by default. Personally, I'd still like to be able to run these fixes and not overwrite the existing font, even with this arg applied.

Reply:
@alerque I foolishly didn't set this PR to a draft. You'll probably uncover a lot of wtf moments for the time being. I always do a cleanup once I've got a draft down. I'd be grateful for any api suggestions/help though since you're a prominent user.

Reply:
Negative on `--force`, that is a very common idiom for for an action that is likely destructive.

How about `--include-source-fixes` or something similarly verbose?

(You know you can convert PR's to Draft mode after the fact right? It's in the "Reviewers" section of the sidebar.)

Reply:
`include-source-fixes` is much better. If we can't think of anything better, we'll roll with it. Thanks for the draft convert as well!

Reply:
@alerque, slept on it and I think `--include-source-fixes` will do. Thanks.